docs/crates/network.md

```diff
@@ -168,7 +168,7 @@ The `Swarm` struct glues together incoming connections from peers, managing sess
 
 We'll touch more on the `NetworkManager` shortly! It's perhaps the most important struct in this crate.
 
-More information about the discovery task can be found in the [discv4](./discv4.md) chapter.
+More information about the discovery task can be found in the [discv4](https://github.com/ethereum/devp2p/blob/master/discv4.md) chapter.
```

Comment:
We actually removed this section of the book, it might be better to just remove these lines because the reference to the discv4 docs does not really make sense in context